zebra/zebra_srv6_vty.c

```diff
@@ -941,11 +998,150 @@ DEFPY(no_srv6_sid_format_explicit,
 	return CMD_SUCCESS;
 }
 
+
+DEFPY(locator_sid, locator_sid_cmd,
+      "sid WORD behavior <uN | uDT4 vrf VIEWVRFNAME | uDT6 vrf VIEWVRFNAME | uDT46 vrf VIEWVRFNAME>",
+      "Configure static local sid\n"
+      "Specify SRv6 locator hex sid\n"
+      "Configure behavior\n"
+      "Apply the code to an End.uN SID\n"
+      "Apply the code to an End.uDT4 SID\n"
+      "vrf\n"
+      "vrf\n"
+      "Apply the code to an End.uDT6 SID\n"
+      "vrf\n"
+      "vrf\n"
+      "Apply the code to an End.uDT46 SID\n"
+      "vrf\n"
+      "vrf\n")
+{
+	VTY_DECLVAR_CONTEXT(srv6_locator, locator);
+	struct seg6_sid *srv6_sid = NULL;
+	struct listnode *node = NULL;
+	enum seg6local_action_t sidaction = ZEBRA_SEG6_LOCAL_ACTION_UNSPEC;
+	int idx = 0;
+	char *vrfName = NULL;
+	char *prefix = NULL;
+	int ret = 0;
+	struct prefix_ipv6 ipv6prefix = { 0 };
+	struct zserv *client;
+	struct listnode *client_node;
+	struct vrf *vrf = NULL;
+
+	if (!locator->status_up) {
+		vty_out(vty, "Missing valid prefix.\n");
```

Comment:
The configuration of sids should always be possible.
Adding to this, modifying the prefix should trigger an update on the sid list.
Each SID should have a status that tells if it is available or not.

some cases:
- I create a locator without prefix, and configure the sid list
- I modify the prefix list after having configured the sid list: I don't expect sids to be reconfigured
- I configured the locator in uncompressed mode (non microsid) : I expect mentioned sids will be inactive.